nuxeo-server-web-ui/Jenkinsfiles/server-build-web-ui-release.groovy

```diff
@@ -0,0 +1,119 @@
+/*
+* (C) Copyright 2022 Nuxeo (http://nuxeo.com/) and others.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* Contributors:
+*     Thomas Fowley
+*/
+
+String getMavenArgs() {
+  def args = '-B -nsu -Dnuxeo.skip.enforcer=true -P-nexus,nexus-private'
+  if (!isPullRequest()) {
+    args += ' -Prelease'
+  }
+  return args
+}
+
+void getNuxeoImageVersion() {
+  return sh(returnStdout: true, script: 'mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate -Dexpression=nuxeo.platform.version -q -DforceStdout').trim();
+}
```

Comment:
we will probably get the Nuxeo image version from a job parameter, see some other jobs as example